pkg/environment/config.go

```diff
@@ -36,12 +41,12 @@ func GetConfig() (base Config, err error) {
 	if err != nil {
 		return
 	}
-	base, err = getConfig(env.RunningMode, baseExtendedURL)
+	base, err = getConfig(env.RunningMode, baseExtendedURL, defaultHttpTimeout)
```

Comment:
This is good solution of course. But I am wondering if in the future we needed to customize more of the client behaviour that the argument list will keep growing. May be there is a better way to do this 🤷🏼‍♂️